addons/snailmail/models/snailmail_letter.py

```diff
@@ -460,25 +464,33 @@ def _append_cover_page(self, invoice_bin: bytes):
 
         cover_buf = io.BytesIO()
         canvas = Canvas(cover_buf, pagesize=A4)
+        font_path = get_module_resource('web', 'static/fonts/lato/Lato-Reg-webfont.ttf')
+        pdfmetrics.registerFont(TTFont("Lato", font_path))
         styles = getSampleStyleSheet()
+        addressStyle = ParagraphStyle(
```

Comment:
Use snake_case for variable names.
```suggestion
        address_style = ParagraphStyle(
```